lib/sidekiq/cron/poller.rb

```diff
@@ -5,20 +5,17 @@
 
 module Sidekiq
   module Cron
-    POLL_INTERVAL = Sidekiq::Options[:average_scheduled_poll_interval] || 30
+    POLL_INTERVAL = 30
 
     # The Poller checks Redis every N seconds for sheduled cron jobs.
     class Poller < Sidekiq::Scheduled::Poller
-      def initialize
-        Sidekiq.configure_server do
-          Sidekiq::Options[:poll_interval_average] = POLL_INTERVAL
-        end
-
+      def initialize(options)
```

Comment:
I would like to suggest to make this parameter optional to make it a little more clean.

```suggestion
      def initialize(options = {})
```